src/index.html

```diff
@@ -5,9 +5,371 @@
+          <img
+            src="images/photos/6.jpg"
+            alt="Marina Palms bay"
+            class="gallery__img"
+          >
+        </div>
+        <div class="gallery__text">
+          <a
+            href="https://www.google.com/maps/place/Faena+House/@25.8078488,-80.1254189,17z/data=!3m1!4b1!4m5!3m4!1s0x88d9b366e4d4a9cb:0x536354b4fd67d316!8m2!3d25.8078488!4d-80.1232302"
+            class="gallery__place"
+          >
+            Faena House /
+          </a>
+          <p class="gallery__details"> Miami Beach, FL 33140</p>
+        </div>
+        <button class="gallery__button button">View all</button>
+      </section>
+      <!--Consultation-->
+      <section class="consultation page__consultation section" id="consultation">
+        <h2 class="consultation__name name">
+          Consultation
+        </h2>
+        <img
+          src="images/crown.svg"
+          alt="logo"
+          class="consultation__logo"
+        >
+        <div class="consultation__main">
+          <div class="consultation__content">
+            <h4 class="consultation__heading">
+              How long ago did you make <br> yourself a gift?
+            </h4>
+            <p class="consultation__text">
+              The procedural change mezzo forte starts izoritmichesky
+              pickup at these moments stop LA Mazel and VA Tsukkerman in his
+              "Analysis of musical works." Pointillism, which
+              originated in the music of the early twentieth
+              century, microforms, found a distant historical.
+            </p>
+          </div>
+          <div class="consultation__right">
+            <form
+              action="aionovaionova@gmail.com"
+              method="POST"
+              class=" consultation__form form"
+            >
+              <input
+                type="text"
+                placeholder="Name"
+                required
+                name="name"
+                class="form__input"
+              >
+              <input
+                type="tel"
+                placeholder="Phone"
+                name="phone"
+                class="form__input"
+              >
+              <input
+                type="email"
+                placeholder="Email"
+                name="email"
+                class="form__input"
+              >
+              <button type="submit" class="form__button button button--blue">
+                Consultation
+              </button>
+            </form>
+          </div>
+        </div>
+      </section>
+      <!--Reviews-->
+      <section class="reviews page__reviews section" id="reviews">
+        <h2 class="reviews__name name">Testimonials</h2>
+        <img
+          src="images/crown.svg"
+          alt="logo"
+          class="reviews__logo logo"
+        >
+        <div class="reviews__cards">
+          <div class="reviews__card card">
+            <img
+              src="images/people/Ellipse1.png"
+              alt="Leslie"
```

Comment:
І тут на майбутнє здається краще задавати загальний опис до картинки не привязуючись на 100% до змісту, до прикладу
```suggestion
              alt="a photo portrait a someone of the company team"
```
Чи в такому дусі.. Також би хотів щоб ментор прокоментував, бо не впевнений чи я правий..)

Reply:
І тоді не треба буде чи в кожному екземплярі вірний alt бо він загальний, а бо тут ти якраз пропустила, і не змінила 2 наступні..